.travis.yml

```diff
@@ -17,14 +17,17 @@ env:
   - THREADSAFE=true DB=postgresql
   - THREADSAFE=false DB=postgresql
 gemfile:
+  - gemfiles/Gemfile.rails-5.0-stable
   - gemfiles/Gemfile.rails-4.2-stable
   - gemfiles/Gemfile.rails-4.1-stable
   - gemfiles/Gemfile.rails-4.0-stable
   - gemfiles/Gemfile.rails-3.2-stable
   - Gemfile
 matrix:
   allow_failures:
-    - gemfile: Gemfile # Not supporting rails 5 at the moment
+    - rvm: 2.0.0
+    - rvm: 2.1.0
```

Comment:
Good point, I haven't looked a the travis doc in a while but do you know if there's a way to allow failure for specific values of the build matrix?

In this case it would be great to allow failures for ruby 2.0.0 and 2.1.0 only for rails 5 but not for other versions